assets/js/blocks/cart-checkout/cart/full-cart/style.scss

```diff
@@ -251,7 +251,7 @@ table.wc-block-cart-items {
 		}
 	}
 	.wc-block-components-radio-control__option {
-		padding-left: $gap-large;
+		padding-left: 40px;
```

Comment:
I think we can remove this ruleset completely now that the [default padding](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/3597/files#diff-29bc3ff26316a7c413ce211149319161bc52bf9d4c0d36afba3725ed13a25ec0R5) is the same:

```diff
-	.wc-block-components-radio-control__option {
-		padding-left: 40px;
-	}
```